lib/services/bitcoind.js

```diff
@@ -333,18 +333,19 @@ Bitcoin.prototype._loadSpawnConfiguration = function(node) {
 
   $.checkArgument(this.options.spawn, 'Please specify "spawn" in bitcoind config options');
   $.checkArgument(this.options.spawn.datadir, 'Please specify "spawn.datadir" in bitcoind config options');
+  $.checkArgument(this.options.spawn.configPath, 'Please specify "spawn.configPath" in bitcoind config options');
   $.checkArgument(this.options.spawn.exec, 'Please specify "spawn.exec" in bitcoind config options');
 
   this._expandRelativeDatadir();
 
   var spawnOptions = this.options.spawn;
-  var configPath = path.resolve(spawnOptions.datadir, './bitcoin.conf');
+  var configPath = path.resolve(spawnOptions.configPath);
 
   log.info('Using bitcoin config file:', configPath);
 
   this.spawn = {};
-  this.spawn.datadir = this.options.spawn.datadir;
```

Comment:
did this one not work either?

Reply:
no that works, this code is a little messy and could be cleaner. i tried not to touch too much. :)